app/metrics/CloudWatch.scala

```diff
@@ -51,33 +60,43 @@ object CloudWatch {
       }
   }
 
-  def getOldCountRequest: GetMetricStatisticsRequest = {
+  private def getRequest(metricName: String): GetMetricStatisticsRequest = {
     val now = DateTime.now(DateTimeZone.UTC)
     new GetMetricStatisticsRequest()
       .withNamespace(namespace)
       .withMetricName(metricName)
       .withDimensions(dimensions.asJava)
       .withPeriod(60 * 60 * 24)  // 1 day (24 hrs)
-      .withStartTime(now.minusDays(30).toDate)
+      .withStartTime(now.minusDays(60).toDate)
       .withEndTime(now.toDate)
       .withStatistics(Statistic.Maximum)
   }
 
-  def extractCountRequestData(result: GetMetricStatisticsResult): List[(DateTime, Double)] = {
+  private[metrics] def extractDataFromResult(result: GetMetricStatisticsResult): List[(DateTime, Double)] = {
     result.getDatapoints.asScala.toList.map { dp =>
       new DateTime(dp.getTimestamp) -> dp.getMaximum.toDouble
     }.sortBy(_._1.getMillis)
   }
 
-  def getOldCountData(client: AmazonCloudWatchAsyncClient)(implicit executionContext: ExecutionContext): Future[List[(DateTime, Double)]] = {
-    awsToScala(client.getMetricStatisticsAsync)(getOldCountRequest).map(extractCountRequestData)
+  private def getWithRequest(request: GetMetricStatisticsRequest)(implicit executionContext: ExecutionContext): Future[List[(DateTime, Double)]] = {
+    awsToScala(client.getMetricStatisticsAsync)(request).map(extractDataFromResult)
+  }
+
+  private def putWithRequest(request: PutMetricDataRequest) = {
+    awsToScala(client.putMetricDataAsync)(request)
   }
 
-  def attemptOldCountData(client: AmazonCloudWatchAsyncClient)(implicit executionContext: ExecutionContext): Attempt[Option[List[(DateTime, Double)]]] = {
-    val tmp: Future[Either[AMIableErrors, Option[List[(DateTime, Double)]]]] = getOldCountData(client).map(ds => Right(Some(ds)))
+  private def attemptWithRequest(request: GetMetricStatisticsRequest)(implicit executionContext: ExecutionContext): Attempt[Option[List[(DateTime, Double)]]] = {
+    val tmp: Future[Either[AMIableErrors, Option[List[(DateTime, Double)]]]] = getWithRequest(request).map(ds => Right(Some(ds)))
```

Comment:
this type-annotated `tmp` is a mistake on my part. It's what I do while I'm still working out the implementation, I should have inlined it when I'd worked it out. Be good to clear that up here since this is changing anyway.